### PR TITLE
[WIP] moose-test auto scaling

### PR DESCRIPTION
**Pull request by lindsayad**
Converts the moose test suite over to using automatic scaling by default.

This process unearthed a coupled of bugs with respect to time integration.

### CODE REVIEW

File: test/src/base/MooseTestApp.C [hunk 1]
```diff
@@ -31,7 +31,7 @@ validParams<MooseTestApp>()
                                    "--disallow-test-objects",
                                    false,
                                    "Don't register test objects and syntax");
-  params.set<bool>("automatic_automatic_scaling") = false;
+  params.set<bool>("automatic_automatic_scaling") = true;
```
Comment (permcody) [inline test/src/base/MooseTestApp.C]: image

BTW - How does this work? I didn't see where you are retrieving this parameter. Is it a typo?

Comment (permcody) [reply test/src/base/MooseTestApp.C]: Oh I see - it's already in the codebase, just not this PR.

**Comment by tophmatthews** [comment]
ping @gardnerru

**Comment by tophmatthews** [comment]
@lindsayad This PR seems to take scaling to a more appropriate level of vetting, are there plans to get it in, and move down the line to other modules? I haven't had time to really start applying this and mortar to bison calcs, but plan to in the next couple of weeks.

**Comment by lindsayad** [comment]
Yes, I do plan to get it in. But there is at least one failing test that I
want to fix up using https://github.com/idaholab/moose/pull/13769, so this
has to wait until I'm done with that PR

**Comment by tophmatthews** [comment]
Great! Just wanted to make sure the stalebot dosen't bite this to oblivion

**Comment by tophmatthews** [comment]
> Great! Just wanted to make sure the stalebot dosen't bite this to oblivion

💀

**Comment by tophmatthews** [comment]
What happened with this one @lindsayad. Hopefully it’s redundant now, but I’m worried about the “unearthed a couple of bugs”

**Comment by lindsayad** [comment]
If all you use is the default time integrator then there are no concerns

**Comment by lindsayad** [comment]
Keeping this as a reminder. A lot of the framework work from this is already in, but it would be nice to get all the tests working (which this does except for one at last check). Most important thing that remains is to do convergence studies with automatic scaling for all the different time integrators

**Comment by lindsayad** [comment]
Still on my todo list...

**Comment by permcody** [comment]
Uh huh.... sure....

**Comment by lindsayad** [comment]
It's 90% done

**Comment by permcody** [comment]
Sounds like something @Derek Gaston <friedmud@gmail.com> would say.

**Comment by lindsayad** [comment]
Closing this; I've got a ticket open for it

**Pull request closed without merging**